mlrun/run.py

```diff
@@ -153,22 +154,25 @@ def run_local(
         if len(sp) > 1:
             args = args or []
             args = sp[1:] + args
+        name = name or pathlib.Path(command).stem
 
     meta = BaseMetadata(name, project=project, tag=tag)
     command, runtime = _load_func_code(command, workdir, secrets=secrets, name=name)
 
     if runtime:
         handler = handler or get_in(runtime, "spec.default_handler", "")
         meta = BaseMetadata.from_dict(runtime["metadata"])
-        meta.name = name or meta.name
```

Comment:
Why you remove this ?

Reply:
we want the local function to have the exact same name as the job function (so in UI its the same group)